tests/test_integration_all_rust.py

```diff
@@ -116,7 +115,7 @@ def get_free_port():
 For local test debugging, set `AUTOPUSH_MP_CONFIG=_url_` to override
 creation of the local server.
 """
-MEGAPHONE_CONFIG = copy.deepcopy(CONNECTION_CONFIG)
+MEGAPHONE_CONFIG:dict[str, str|int|float|None] = copy.deepcopy(CONNECTION_CONFIG)
```

Comment:
I was using black to do the formatting fixups, since I thought that was the new standard. (specifically `black -l 78`)

Reply:
switched to using `dict`.

Reply:
black's definitely the new standard but it can format significantly differently than previous formatters IMO -- so I suggested our original flake8 to reduce the diff size (ideally we would have applied black in a separate easy to read PR/commit).

IMO if you're switching to `dict` you should also switch from `typing.List` to `list` as well, but this is just a minor nit.

Reply:
Possibly. I also figured since this is the initial add, we should format it appropriately, since this technically isn't a diff, but including the new python3 script.